lib/elixir/lib/kernel.ex

```diff
@@ -525,15 +526,16 @@ defmodule Kernel do
       :b
 
   """
-  @spec max(first, second) :: (first | second) when first: term, second: term
-  def max(first, second) do
-    :erlang.max(first, second)
+  @spec max(left, right) :: (left | right) when left: term, right: term
+  def max(left, right) do
+    :erlang.max(left, right)
```

Comment:
I agree they are not great but the reason I don't think `value` is an improvement is because we, of course, already know they are values.